Makefile.am

```diff
@@ -47,6 +47,7 @@ bin_PROGRAMS = \
     tools/tpm2_dictionarylockout \
     tools/tpm2_duplicate \
     tools/tpm2_getcap \
+    tools/tpm2_getname \
```

Comment:
tpm2_readpublic doesn't work?

Reply:
Used tpm2_readpublic for non-simulator TPMs which do not support Tpm2_encryptdecrypt now.